src/sandstorm/test-app/test-app.c++

```diff
@@ -204,8 +241,8 @@ public:
 
     // The `CLIENT` side of a `capnp::TwoPartyVatNetwork` does not serve its bootstrap capability
     // until it has initiated a request for the bootstrap capability of the `SERVER` side.
-    // Therefore, we need to restore the supervisor's `SandstormApi` capability, even if we are not
-    // going to use it.
+  // Therefore, we need to restore the supervisor's `SandstormApi` capability, even if we are not
```

Comment:
Seems like an inadvertent indentation change here.

Reply:
Yes, but this whole comment went away later, since we actually use it now.